workspace-manager-clienttests/src/main/java/scripts/testscripts/DataReferenceLifecycle.java

```diff
@@ -0,0 +1,53 @@
+package scripts.testscripts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import bio.terra.workspace.model.CreateDataReferenceRequestBody;
+import bio.terra.workspace.model.DataReferenceDescription;
+import bio.terra.workspace.model.ReferenceTypeEnum;
+import scripts.utils.WorkspaceFixtureTestScriptBase;
+import scripts.utils.ClientTestUtils;
+
+public class DataReferenceLifecycle extends WorkspaceFixtureTestScriptBase {
```

Comment:
This one is almost identical to `CreateGetDeleteDataReference` in #183 - we could really use either as long as we include them in the integration test suite.

Reply:
I think this one has some extra assertions.